src/pages/resultsView/mutation/MutationStatusSelector.tsx

```diff
@@ -88,7 +88,11 @@ export default class MutationStatusSelector extends React.Component<
 
     private get germlinePlaceholder() {
         return (
-            <div data-test="germlineMutationRate" className="invisible">
+            <div
+                data-test="germlineMutationRate"
+                className="invisible"
+                style={{ display: 'none' }}
```

Comment:
As far as I remember we didn't want `{ display: 'none' }` because we wanted to render certain elements always at a fixed location. But now we are switching to a compact view, probably it's okay to do it this way.